gradle/libs.versions.toml

```diff
@@ -19,12 +19,19 @@ kotlin = "1.6.10"
 lifecycle = "2.4.1"
 navigation = "2.4.2"
 accompanist = "0.23.1"
+googleServices = "4.3.10"
```

Comment:
it will be better to leave sorted as previously)

Reply:
done